Paws4Me/Paws4Me/Controllers/Logic/SinglePetViewModel.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  SinglePetViewModel.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/15.
+//
+
+import Foundation
```

Comment:
You can remove this file for now until you need to use it 😄 remember we don't want to be actively adding to this PR is has been open for quite some time already so adding more code will make this review take longer.

Reply:
Understood.